src/utils/querybuilder.util.js

```diff
@@ -519,7 +519,7 @@ const quantityQueryBuilder = function ({ target, field }) {
         }
     } else {
         // no prefixes
-        qB[`${field}.value`] = Number(num);
+        qB[`${field}.value`] = { $gt: 0.99 * num, $lt: 1.01 * num };
```

Comment:
1. The range for Quantity type number should be calculated with taking precision into account as described in https://hl7.org/fhir/R4B/search.html#number
For example in case of <base>/Observation?value-quantity=100.00
we are getting range as {'$gt':99,'$lt':101}}
but it should be {'$gt':99.995,'$lt':100.005}}

2. Also the range need to be implemented in case of 'ne' prefix too.

Reply:
Updated with new range calculations.
For 'ne', is OK, or should switch polarity and use an $or?